### DISCUSSION TITLE
Rename organization

### DISCUSSION THREAD

**Discussion by akouete-kpakpo** [ML-for-B-E/.github#12] [Ideas]
@BWhannou maybe we should rename the organization "EEIA" and add our logo

![image](https://user-images.githubusercontent.com/26568112/185803796-42b2fab0-6f3a-4403-802e-e8d0be634fb0.png)

What do you think?

**Comment by akouete-kpakpo** [comment]
Or "benin-excellence" with logo

![image](https://user-images.githubusercontent.com/26568112/185803842-6bd003af-2e9e-427d-9623-b7049548fd4d.png)

**Comment by BWhannou** [comment]
I made some changes. However I see there are some side effects when we change names, see below:
![image](https://user-images.githubusercontent.com/9529168/185805213-dc334b70-102e-4fa3-9ca2-e64cf984aaa1.png)

So I didn't change it yet

**Reply by akouete-kpakpo** [reply]
hi @BWhannou . yes on our side, it will imply updating our  local repositories config especially the remote url with the new one after renaming. Or simpler, delete the local repo and reclone it.

**Reply by akouete-kpakpo** [reply]
probably we should make a general communication so that everyone is aware of this change, before doing the renaming.
